packages/carbon-graphs/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "@cerner/carbon-graphs",
+  "main": "lib/Graphs.js",
+  "private": true,
+  "version": "0.0.0",
+  "description": "A graphing library built using d3 based on Cerner design standards",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-graphs.git",
+    "directory": "packages/carbon-graphs"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "carbon-graphs",
+    "carbon",
+    "graphs",
+    "UI",
+    "d3",
+    "d3-module",
+    "graphing"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-graphs/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-graphs#readme",
+  "files": [
+    "dist",
+    "lib",
+    "README.md"
+  ],
+  "prettier": {
+    "arrowParens": "always",
+    "bracketSpacing": true,
+    "semi": true,
+    "tabWidth": 4,
+    "trailingComma": "none",
+    "overrides": [
+      {
+        "files": "package.json",
+        "options": {
+          "tabWidth": 2,
+          "printWidth": 250
+        }
+      }
+    ]
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "autoprefixer": "^9.8.4",
+    "classnames": "^2.2.5",
+    "d3": "^5.16.0",
+    "jasmine-core": "^3.5.0",
+    "prop-types": "^15.5.8",
+    "terra-theme-context": "^1.0.0"
+  },
+  "peerDependencies": {
+    "react": "^16.8.5",
+    "react-dom": "^16.8.5"
+  },
+  "scripts": {
+    "compile": "babel --root-mode upward src --out-dir lib --copy-files",
+    "lint": "npm run lint:js && npm run lint:scss",
+    "lint:js": "eslint --ext .js,.jsx . --ignore-path ../../.eslintignore",
+    "lint:scss": "stylelint src/**/*.scss",
+    "precompile": "rm -rf lib",
+    "test": "npm run lint && npm run jest && npm run wdio",
+    "jest": "jest --config ../../jest.config.js",
+    "wdio-default": "cd ../.. && npx wdio",
+    "wdio-lowlight": "cd ../.. && THEME=clinical-lowlight-theme npx wdio",
+    "wdio-fusion": "cd ../.. && THEME=orion-fusion-theme npx wdio",
+    "wdio": "npm run wdio-default && npm run wdio-lowlight && npm run wdio-fusion",
```

Comment:
You won't be running wdio on this package directly. WDIO tests should be run through the terra-graphs package. You can remove these.

Reply:
5865a2ffd495086785b7a137553f7e5f3b3b25c0